### DISCUSSION TITLE
The easiest way to play USC

### DISCUSSION THREAD

**Discussion by ogami1972** [Drewol/unnamed-sdvx-clone#597] [General]
Hello!

I'd like to quickly document what I believe the easiest path to play USC. This setup should work on pretty much any device supported by the software described. This would include Various x86 pc's, Macs, Rpi's and a whole lot more, provided the platform supports the build.

Supplies: 

- 256gb+ flash drive
- A controller of some sort for menu navigation in Batocera
- PC or Mac with the ability to burn an image to a drive and boot to USB
- Internet Access
- A bit of PC/super-novice linux knowledge
- Free Time (at least a few hours)

1. Visit [Batocera.com](https://batocera.org/download) and download the appropriate build for your device. Put simply, Batocera is a frontend for gaming emulation. On x86 builds, Windows games are supported. I believe this to be the case on many other builds as well, but your mileage may vary. A more detailed explanation is beyond the scope of this guide, so please do your research.
2. Burn the image to your flash drive using Balena or Rufus or Gnome Disks or whatever. Feel free to ask questions if you don't know what this means.
3. Boot to your batocera drive and allow Batocera to go through it's initial setup (resizing partitions and such). Once you are at the main menu, setup your controller as necessary. I am assuming you will use some sort of gamepad to navigate Batocera, although you can use a keyboard in the game itself, so don't worry too much what kind of gamepad you use. If you have a proper controller, say so and I can give advice on configuration.
4. Now restart your machine and return to your normal OS. [Download the latest build of the game](https://drewol.me/Downloads/Game.zip). Place this on your batocera drive at the following similar path : /userdata/ROMS/Windows . This can be done by accessing the Batocera drive on your normal OS and simply drag and drop.
5. Boot back to your Batocera drive and reload the games list. You should now see a listing for "usc-game" or similar in your "Windows" section of Batocera. Launch it. You will see it what looks like the Windows BSOD, but it's just Batocera setting up the game. After a moment, the game should launch normally. If it doesn't, stop here and say so and we can figure it out.
6. Exit Batocera and back to your normal OS. Obtain any charts you want from the normal locations. These are put on your Batocera drive, something like "/userdata/ROMS/Windows/Game/songs". Likewise, any skins you want should be obtained and placed in "../skins".

That's pretty much it. Follow these steps, and, provided it works, lol, you will have a fully portable USC drive. Take it anywhere, boot to it and blow people's minds. :dancers: 

Feel free to ask questions...

p.s. I actually "installed" my Batocera to a proper SSD, then set USC to launch on boot, so the pc appears to be a "USC Machine" ;)

**Comment by FoxMadeline** [comment]
this sounds like a neat idea!
i may try this one day, thank you for the guide
